src/components/command-search.tsx

```
@@ -0,0 +1,31 @@
+"use client";
+
+import { Command } from "cmdk"
+
+interface Props {
+  placeholder: string;
+  data: {
+    
+  }[];
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)